### PR TITLE
Added `build/` to `.gitignore`

### PR DESCRIPTION
**Pull request by larsewi**
# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [x] Make sure that all commits follow the conventional commit [specification](https://www.github.com/mendersoftware/mendertesting/commitlint/grammar.md) for the Mender project.

The majority of our contributions are fixes, which means your commit should have
the form below:

```
fix: <SHORT DESCRIPTION OF FIX>

<OPTIONAL LONGER DESCRIPTION>

Changelog: <USER-FRIENDLY-CHANGE-DESCRIPTION> or <None>
Ticket: <TICKET NUMBER> or <None>
```

- [x] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Added `build/` to `.gitignore` in order to provide the means for my favourite command `git add .`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,3 +59,6 @@ config.h
 !/cmake/*.cmake
 /Testing
 /install_manifest*.txt
+
+# VS Code
+.vscode/
```
[the remark this answers is not in the document]
Comment (larsewi) [reply .gitignore]: Good point! I will drop the commit 😉

**Review by oleorhagen — COMMENTED** [review]
Fine with me.

I think I would use the `build:` prefix in the commit though.

Have a look [here](https://github.com/mendersoftware/mendertesting/blob/master/commitlint/grammar.md) if you are unfamiliar :)

**Review by oleorhagen — APPROVED** [review]

**Comment by larsewi** [comment]
Officially a Mender client contributor 🤣

**Comment by oleorhagen** [comment]
> Officially a Mender client contributor rofl

Congrats :smile_cat:

**Pull request merged**